version-catalog/src/test/kotlin/io/spine/internal/catalog/entry/CatalogEntryLoadTest.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2022, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.internal.catalog.entry
+
+import io.spine.internal.catalog.CatalogEntry
+import io.spine.internal.catalog.VersionCatalogEntryLoader
+import org.junit.jupiter.api.Test
+
+/**
+ * It is used only for development purposes.
+ */
+internal class CatalogEntryLoadTest {
+
+    private val entriesUnderTest = setOf(
+        AnimalSniffer::class.java,
+//        ApacheCommons::class.java,
+//        KotlinX::class.java,
+    )
+
+    @Test
+    fun shouldLoadEntries() {
+//        entriesUnderTest.forEach { it.load() }
+    }
+
+    private fun Class<out CatalogEntry>.load() {
+//        println()
```

Comment:
Why do we need these commented blocks?

Reply:
The documentation says that this class is used for the "development purposes", which hints on some special kind of purposes, but it's not clear which.